docs/monitors/logstash.md

```diff
@@ -31,10 +31,16 @@ Configuration](../monitor-config.md#common-configuration).**
 
 | Config option | Required | Type | Description |
 | --- | --- | --- | --- |
+| `httpTimeout` | no | `int64` | HTTP timeout duration for both read and writes. This should be a duration string that is accepted by https://golang.org/pkg/time/#ParseDuration (**default:** `10s`) |
```

Comment:
If somebody changes `timeoutSeconds: 5` to `httpTimeout: 5`, what will it be interpreted as?  Or will it error?

Reply:
Unfortunately it gets interpreted as nanoseconds since time.Duration is an `int64`.

Reply:
I'm adding a custom time.Duration for it so we can enforce it being a string. I can easily see people (or me..) forgetting to specify the unit and have it silently using things like 5ns as the timeout.

Reply:
Ok, yeah I think having the default interpretation of integers as seconds is the best way to go.

Reply:
I've done the PR for this in https://github.com/signalfx/signalfx-agent/pull/1168 as we have existing time.Duration usage in master that this would benefit from.